lib/pbench/test/unit/server/conftest.py

```diff
@@ -12,13 +11,13 @@
 [pbench-server]
 pbench-top-dir = {TMP}/srv/pbench
 
-[elasticsearch]
-host = elasticsearch.example.com
-port = 7080
+[elasticsearch]	
```

Comment:
Did you mean to add trailing whitespace on these lines? How'd it get there?

Reply:
Ahh yeah I remove them, I was playing around with it and it got added